types/atto_fil.go

```diff
@@ -71,37 +58,29 @@ func (z AttoFIL) MarshalJSON() ([]byte, error) {
 // AttoFIL represents a signed multi-precision integer quantity of
 // attofilecoin (atto is metric for 10**-18). The zero value for
 // AttoFIL represents the value 0.
-type AttoFIL struct{ val *big.Int }
+type AttoFIL struct{ val big.Int }
```

Comment:
Copy your words describing the tradeoff between embedding the bigint directly here, vs as a pointer, from the PR description. Save the next person the effort of retracing your steps.